AppCenterDataStorage/AppCenterDataStorageTests/MSTokenTests.m

```diff
@@ -21,6 +37,194 @@ - (void)tearDown {
   [super tearDown];
 }
 
-- (void)testGetToken {
+- (void)testGetTokenResult {
+
+  // When
+  MSTokenResult *result = [[MSTokenResult alloc] initWithPartition:partitionName
+                                                         dbAccount:dbAccount
+                                                            dbName:dbName
+                                                  dbCollectionName:dbCollectionName
+                                                             token:token
+                                                            status:status
+                                                         expiresOn:expiresOn];
+
+  // Then
+  [self compareWithTokenObject:result];
+
+  // When
+  NSString *resultString = [result serializeToString];
+  NSDictionary *resultDic = [self deserializeDataString:resultString];
+
+  // Then
+  [self compareWithDictionary:resultDic];
+}
+
+- (void)testGetTokenResultWithString {
+
+  // If
+  NSData *tokenData = [NSJSONSerialization dataWithJSONObject:[self getDefaultTokenData] options:NSJSONWritingPrettyPrinted error:nil];
+  NSString *tokenString = [[NSString alloc] initWithData:tokenData encoding:NSUTF8StringEncoding];
+
+  // When
+  MSTokenResult *result = [[MSTokenResult alloc] initWithString:tokenString];
+
+  // Then
+  [self compareWithTokenObject:result];
+
+  // When
+  NSString *resultString = [result serializeToString];
+
```

Comment:
Remove extra new line

Reply:
There are a lot of extra new lines, I won't comment for each one but please check for them.